avm/res/cache/redis-enterprise/tests/e2e/active-geo-replication/main.test.bicep

```diff
@@ -0,0 +1,76 @@
+targetScope = 'subscription'
+
+metadata name = 'Active geo-replication'
+metadata description = 'This instance deploys the module with active geo-replication enabled.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-cache-redisenterprise-${serviceShort}-rg'
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'creagr'
+
+@description('Optional. A token to inject into the name of each resource. This value can be automatically injected by the CI.')
+param namePrefix string = '#_namePrefix_#'
+
+@description('Optional. The name of the geo-replication group.')
+param geoReplicationGroupName string = 'geo-replication-group'
```

Comment:
A last property that should be moved to the `dependencies.bicep` as it's not passed in by the CI. As far as I'm concerned, it coult potentially even be specified directly where it is used and then returned by the `dependencies.bicep` file like `output geoReplicationResourceGroupName string = redisCluster::redisDatabase.properties.geoReplication.groupNickname` and be referenced as such.

Reply:
Updated